site/introducing-weave-scope.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Introducing Weave Scope
+menu_order: 10
+---
+
+
+Weave Scope is a browser-based infrastructure, app diagnostics and container visualization tool. It provides a top down view into your app as well as your entire infrastructure, and allows you to diagnose any problems with your distributed containerized app, in real time, as it being deployed to a cloud provider.  
```

Comment:
The tag line we've been using is "Weave Scope: Monitoring, visualisation & management for Docker & Kubernetes".  How does this introduction fit with that?